src/lightning/pytorch/loggers/mlflow.py

```diff
@@ -109,6 +109,12 @@ def any_lightning_module_function_or_hook(self):
         ModuleNotFoundError:
             If required MLFlow package is not installed on the device.
 
+    Note:
+        As of vX.XX, MLFlowLogger will skip logging any metric (same name and step)
```

Comment:
No need to mention what version this was changed in
```suggestion
        MLFlowLogger will skip logging any metric (same name and step)
```